xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
Thanks for the example! Understand completely now. Sorry for the trouble.

The "turnout followed by another turnout" case wasn't properly handled, because the code wasn't being allowed to consider a DivergingClear as an option when the next signal was showing DivergingClear.  This makes that possible.